doc/overview.md

```diff
@@ -0,0 +1,67 @@
+# Overview
+
+As enterprises migrate from traditional VM infrastructures to more agile
+microservice-based deployments on container platforms, there is a need for
+a simplifying technology that can take care of common cross-cutting
+capabilities around service communication and management such as secure
+interconnect, service discovery & load balancing, staged rollouts, A/B
+testing, intelligent rate limiting, authentication, access control,
+monitoring, logging, etc.
+
+Broadly speaking, an _open, platform independent service mesh_ that
```

Comment:
Suggestion

Istio is an _open platform-independent service mesh_ that takes care of traffic management, policy enforcement, rate limiting, and telemetry collection. It allows developers[....] And it simplifies the operator's job